src/ui/components/Events/Events.css

```diff
@@ -0,0 +1,6 @@
+.current-time {
```

Comment:
I went off-tailwind here. This is like a weird decorative div that has very specific display properties. Would this be better inside of a component with inline styles?